.github/workflows/smoke-tests-canaries.yml

```diff
@@ -0,0 +1,17 @@
+name: Smoke Tests - Canaries
+
+# This is required by aws-actions/configure-aws-credentials
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  schedule:
+    - cron: '0 16 * * *' # Everyday 16:00 UTC
+
+jobs:
+  call-smoke-tests:
+    uses: ./.github/workflows/smoke-tests.yml
+    secrets: inherit
+    with:
+      versions: '["latest"]'
```

Comment:
Perhaps we should add `RC` as well to catch any errors before going to production?

Reply:
We could. I was thinking that RC will go through "oncall will click a button to trigger smoke tests", but having an active monitoring here won't hurt I guess.